test/test.js

```diff
@@ -574,6 +575,174 @@ describe('Localize APIs', () => {
+                console.log('result', result);
+                if (err) {
+                    console.log('error in upload Document:' + err);
+                }
+                result.meta.status.should.eql(200);
+                done();
+            });
+        });
+        it('Should Upload fail a translated document', function (done) {
+            const data = {
+                projectKey: project_key,
+                documentId: projectTestData.documentId,
+                fileName: 'uploadDoc',
+                language: '',
+                content: __dirname + '/to-translte.csv',
+
+            };
+            localizeService.documents.createTranslation(data, function (err, result) {
+                if (err) {
+                    err.should.not.be.eql(null);
+                }
+                done();
+            });
+        });
+        it('Should Get a list of documents', function (done) {
+            const data = {
+                projectKey: project_key,
+            };
+            localizeService.documents.getDocuments(data, function (err, result) {
+                if (err) {
+                    console.log('error in get Documents:' + err);
+                }
+                result.data.should.be.an.Array();
+                done();
+            });
+        });
+        it('Should fail to get Get a list of documents without mandatory fields', function (done) {
+            const data = {
+                projectKey: '',
+            };
+            localizeService.documents.getDocuments(data, function (err, result) {
+                if (err) {
+                    err.message.should.eql('Invalid input params');
+                }
+                done();
+            });
+        });
+        it('Should fail to get a translation based on id', function (done) {
+            const data = {
+                projectKey: project_key,
+                documentId: '',
+            };
+            localizeService.documents.downloadDocument(data, function (err, result) {
+                if (err) {
+                    err.message.should.eql('Invalid input params');
+                }
+                done();
+            });
+        });
+        it('Should get a Source document based on id', function (done) {
+            const data = {
+                projectKey: project_key,
+                documentId: projectTestData.documentId,
+            };
+            localizeService.documents.downloadDocument(data, function (err, result) {
+                if (err) {
+                    console.log('error in Source document:' + err);
+                }
+                result.should.have.property('dictionary')
+                done();
+            });
+        });
+        it('Should Get a translated document', function (done) {
+            const data = {
+                language: 'zh',
+                projectKey: project_key,
+                documentId: projectTestData.documentId,
```

Comment:
Can we change this to read configured translation document ID `documentId: projectTestData.translationDocumentId,` instead of reading from one document Id used for both source and translation? It would make it easier to run the test cases in one shot